packages/wasm-bundler/index.mjs

```diff
@@ -140,19 +101,16 @@ traverse(ast, {
     enter(path) {
         const { node } = path;
         if (isFunctionDeclaration(node) && node.id.name === 'load') {
-            // console.log({ node });
-            // console.log({ node: node.body.body });
             const newDeclaration = parse(wasmLoadFunc, { sourceType: 'module' });
-            // console.log({newDeclaration: newDeclaration.program.body[0].body.body});
             node.body.body = newDeclaration.program.body[0].body.body
-            // path.replaceWith(newDeclaration);
+        }
+        if (isFunctionDeclaration(node) && node.id.name === 'init') {
+            node.body.body[2] = parse("");
         }
```

Comment:
So the `init()` function is going to be unused completely? It contains a call to `getImports()`, can we do without them?

Reply:
Nope, only one if statment will be removed, which may call `fetch`.
This is fragile to changes in wasm-pack as everything  in this code base